var/spack/repos/builtin/packages/py-hpcbench/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyHpcbench(PythonPackage):
+    """Define and run your benchmarks"""
+
+    homepage = "https://github.com/BlueBrain/hpcbench"
+
+    url      = "https://pypi.io/packages/source/h/hpcbench/hpcbench-0.8.tar.gz"
+    git      = "https://github.com/BlueBrain/hpcbench.git"
+
+    version('develop', branch='master')
+    version('0.8', sha256='120f5b1e6ff05a944b34a910f3099b4b0f50e96c60cf550b8fc6c42f64194697')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-setuptools-scm', type='build')
+    depends_on('py-cached-property')
+    depends_on('py-clustershell')
+    depends_on('py-cookiecutter')
+    depends_on('py-docopt')
+    depends_on('py-elasticsearch@6:6.999')
+    depends_on('py-jinja2')
+    depends_on('py-mock', type='test')
+    depends_on('py-numpy')
+    depends_on('py-pyyaml')
+    depends_on('py-magic')
+    depends_on('py-six')
```

Comment:
Aside from setuptools/setuptools-scm, these dependencies should all be `type=('build', 'run')`. Also, you need to specify which versions to use:
```python
    install_requires=[                                                          
        'cached-property>=1.3.1',                                               
        'clustershell>=1.8',                                                    
        'cookiecutter==1.6.0',                                                  
        'docopt==0.6.2',                                                        
        'elasticsearch>=6.0.0',                                                 
        'jinja2==2.10',                                                         
        'mock==2.0.0',                                                          
        'numpy==1.13.3',                                                        
        'PyYAML>=3.12',                                                         
        'python-magic==0.4.15',                                                 
        'six==1.11',                                                            
    ],         
```
If the required versions are different for `master`, you need to add those lines too.